examples/sara_work_area/genz_117_pali_structure.py

```diff
@@ -0,0 +1,10 @@
+value = int(input("Please enter the number of lines: "))
```

Comment:
Lets add mutiline string 
"""
pali_structure.py says: What is my responsibility
"""